eng/mgmt/automation/generate_data.py

```diff
@@ -141,10 +140,11 @@ def generate(
             logging.error('[GENERATE] Autorest fail')
             return False
 
-        set_or_increase_version(sdk_root, GROUP_ID, module)
+        set_or_default_version(sdk_root, GROUP_ID, module)
         update_service_ci_and_pom(sdk_root, service, GROUP_ID, module)
         update_root_pom(sdk_root, service)
-        update_version(sdk_root, output_dir)
+        # skip version update script, as current automation does not support automatic version increment
+        # update_version(sdk_root, output_dir)
```

Comment:
Did you consider adding the support to the script instead of having more versioning logic outside of the versioning script?

Reply:
This line just called the `update_versions.py`. So the versioning script does do all the versioning stuff.

The reason I disable this in the PR is that it appears take very long (5min to 10min plus) when running in docker (I guess that disk system of docker is not efficient).
And at least for now, data-plane tooling does not need to automatically update, so I skipped this line.